client/index.html

```diff
@@ -10,13 +10,11 @@
   <body>
     <h1>Courageous Trapeze</h1>
     <div ng-view></div>
-    <script src="lib/angular/angular.js"></script>
+    <script src="lib/underscore/underscore-min.js"></script>
+    <script src="lib/bluebird/js/browser/bluebird.min.js"></script>
+    <script src="lib/jquery/dist/jquery.min.js"></script>
+    <script src="lib/angular/angular.min.js"></script>
     <script src="lib/angular-route/angular-route.min.js"></script>
-    <script src="app/factories/factories.js"></script>
-    <script src="app/contacts/contacts.js"></script>
-    <script src="app/messages/messages.js"></script>
-    <script src="app/schedule/schedule.js"></script>
-    <script src="app/auth/auth.js"></script>
-    <script src="app/app.js"></script>
```

Comment:
@kevhuang I see you removed the dependencies for all Angular controllers and factories in index.html - where did you reference these files instead?